SensoBackend/Application/Modules/Alerts/Contracts/CreateAlertDto.cs

```diff
@@ -0,0 +1,9 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace SensoBackend.Application.Modules.Alerts.Contracts;
+
+public sealed record CreateAlertDto
+{
+    [Required]
+    public required string Type { get; init; }
```

Comment:
We can only trigger SOS alerts from the app, the rest is triggered from the backend. So we could either:
- validate that `Type` is always equal to `Sos`
- remove the param altogether